### DISCUSSION TITLE
gatsby-plugin-feed: How to get favicon?

### DISCUSSION THREAD

**Discussion by cedricdelpoux** [gatsbyjs/gatsby#35562] [Help]
`gatsby-plugin-feed` generate my feed to `/rss.xml` but when I open it, there is no favicon displayed in the browser tab and ther is no icon when browsing the feed from a RSS reader.

How to do it?

**Comment by PoorTable** [comment]
Was You able to solve it? @cedricdelpoux

**Reply by cedricdelpoux** [reply]
No
